src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Bindings/BlobExtensionConfig.cs

```diff
@@ -198,13 +198,13 @@ private class MultiBlobContext
             }
         }
 
-        private async Task<CloudBlobClient> GetClientAsync(
+        private Task<CloudBlobClient> GetClientAsync(
          BlobAttribute blobAttribute,
          CancellationToken cancellationToken)
         {
             var account = _accountProvider.Get(blobAttribute.Connection, _nameResolver);
             var client = account.CreateCloudBlobClient();
-            return client;
+            return Task.FromResult(client);
```

Comment:
This has already been fixed in latest: https://github.com/Azure/azure-webjobs-sdk/blob/3c3fc7dfa107bd65f1ea4e6d03580de66f404695/src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Config/BlobsExtensionConfigProvider.cs#L294